test/match-orders.test.mo

```diff
@@ -70,11 +64,9 @@ do {
     (80.0, 6000),
   ]);
 
-  let (nAsks, nBids, volume, price) = matchOrders(asks, bids);
-  assert nAsks == 1;
-  assert nBids == 2;
+  let (volume, price) = clearAuction(asks, bids);
   assert volume == 10000;
-  assert price == 90.0;
+  assert price == 0.0;
```

Comment:
How it is possible to get non-zero volume with price 0? Bidders will get tokens for free?

Reply:
Discussed offline